app/views/users/_profile.html.erb

```diff
@@ -0,0 +1,10 @@
+<div class="flex items-center justify-center card bg-white p-6 shadow-md rounded-lg ">
+  <div class="mx-5">
+    <img src="<%= user.photo %>" alt="img" class="h-40 w-50 rounded-full object-cover"/>
+  </div>
+  <div class="w-2/3 pl-4 flex justify-between">
+    <h2 class="text-lg text-center font-medium"><%= user.name %></h2>
+    <span class="text-sm items-end mt-12">Number of posts: <%=user.posts.count%></span>
+  </div>
+</div>
```

Comment:
- [**optional**] Hi @belaymit, I think the main idea behind partials is to avoid repeating the same piece of code. I can still find this same piece of code in the `shared/posts` partial. Lets kindly make sure our codebase is DRY

Great job for using partials :+1: